.gitignore

```diff
@@ -1,2 +1,9 @@
+#ignore all python cached files
 __pycache__/
-.vscode/*
\ No newline at end of file
+
+#ignore all vscode settings
+.vscode/
```

Comment:
Thank you for fixing this. I did not realize that having an asterisk after the file name would cause it to not work.